descriptions.yml

```diff
@@ -4,6 +4,381 @@
+Description of line items in an order.
+
+".components.schemas.OrderLineItem.properties.type": >
+Type refers to the entity type of an item whether it is product or promotion for instance.
+
+".components.schemas.OrderLineItem.properties.customFields": >
+Additional fields that offer a possibility to add own fields for the different program-areas.
+
+".components.schemas.OrderTag.properties.orderId": >
+Unique identity of order.
+
+".components.schemas.OrderTag.properties.orderVersionId": >
+Unique identity of order's version.
+
+".components.schemas.OrderTag.properties.tagId": >
+Unique identity of tag given for an order.
+
+".components.schemas.OrderTransaction.properties.versionId": >
+Unique identity of OrderTransaction's version.
+
+".components.schemas.OrderTransaction.properties.orderId": >
+Unique identity of an order.
+
+".components.schemas.OrderTransaction.properties.orderVersionId": >
+Unique identity of an order's version.
+
+".components.schemas.OrderTransaction.properties.paymentMethodId": >
+Unique identity of payment method.
+
+".components.schemas.OrderTransaction.properties.amount.properties.unitPrice": >
+Price of product per item (where, quantity=1).
+
+".components.schemas.OrderTransaction.properties.amount.properties.totalPrice": >
+Cost of product based on quantity.
+
+".components.schemas.OrderTransaction.properties.amount.properties.quantity": >
+Number of items of each product.
+
+".components.schemas.OrderTransaction.properties.stateId": >
+Unique identity of state.
+
+".components.schemas.OrderTransaction.properties.customFields": >
+Additional fields that offer a possibility to add own fields for the different program-areas.
+
+".components.schemas.PaymentMethod.properties.name": >
+Name of the payment method.
+
+".components.schemas.PaymentMethod.properties.distinguishableName": >
+System identifier name for payment method.
+
+".components.schemas.PaymentMethod.properties.description": >
+A short description about the payment method.
+
+".components.schemas.PaymentMethod.properties.position": >
+The order of the tabs of your defined payment methods in the storefront by entering numerical values like 1,2,3, etc.
+
+".components.schemas.PaymentMethod.properties.active": >
+When set to active, the payment method is available for usage.
+
+".components.schemas.PaymentMethod.properties.afterOrderEnabled": >
+When set to true, customers are redirected to the payment options page to rechoose a payment method on order failure.
+
+".components.schemas.PaymentMethod.properties.customFields": >
+Additional fields that offer a possibility to add own fields for the different program-areas.
+
+".components.schemas.PaymentMethod.properties.mediaId": >
+Unique identity of media.
+
+".components.schemas.PaymentMethod.properties.translated": >
+Contains the translations of all translated fields.
+
+".components.schemas.Plugin.properties.translated": >
+Contains the translations of all translated fields.
+=======
```

Comment:
```suggestion
```
I guess this was an accident?

Reply:
Oh i was resolving conflict and dint notice this.